src/platform/forms-system/src/js/helpers.js

```diff
@@ -298,45 +298,83 @@ function isPartialAddress(data) {
   );
 }
 
+/**
+ * Replaces double-quotes with single.
+ * Replaces non-escaping backslashes with forward-slashes.
+ *
+ * @param {string} string
+ * @returns {string}
+ */
+export const replaceEscapedCharacters = string => {
+  return string
+    .replaceAll('"', "'")
+    .replace(/(?:\r\n|\n\n|\r|\n)/g, '; ')
+    .replace(/(?:\t|\f|\b)/g, '')
+    .replace(/\\(?!(f|n|r|t|[u,U][\d,a-fA-F]{4}))/gm, '/');
+};
+
 /**
  * Create a replacer function for JSON.stringify
  * A replacer function is the second argument to JSON.stringify
  *
- * @param {{
- *  allowPartialAddress?: boolean,
- * }} [options] - Options for the replacer
+ * @param {ReplacerOptions} [options] - Options for the replacer
+ * @returns {(key, value) => any} The replacer function
  */
 export function createStringifyFormReplacer(options) {
```

Comment:
Overall, I like this refactor. I'm not in love with the nested function, as I prefer more pure functions that explicitly define their dependencies/parameters (see my comment below about unintentionally creating a closure). But I can work with this. As best as I can understand what is going on here, this feels cleaner and easier to follow.

Reply:
Yeah, I was entirely comfortable with it, but I was trying to disrupt as little as possible. But it is confusing and not best practice. I've extracted both `createStringifyFormReplacer` and `replaceEscapedCharacters` from the `helpers.js` file and made them utility functions.